DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -756,7 +756,15 @@ public class DcChat {
             }
         }
         return nil
-        }()
+	}()
+
+	public var profileImageURL: URL? {
+		guard let cString = dc_chat_get_profile_image(chatPointer) else { return nil }
```

Comment:
somehow this alignment is broken - on my machine it is actually alright? Not sure what's wrong here?

Reply:
seems as if you're using a _tab_ instead of _4 spaces_ for indentation. tabs are rendered as 8 spaces on github - and probably as 4 spaces on your machine.

just use spaces as elsewhere in the file and things should be fine :)